packages/client/src/i18n/it/manage-users.ts

```diff
@@ -144,6 +148,7 @@ export default {
       "Non è stato possibile segnalare il problema per tutte le copie.",
     returnAndDonate: "Restituisci contanti e dona libri",
```

Comment:
Why isn't there the amount to return here? I guess we should add it

@NiccoloMenozzi this seems something that has been decided into design phase, as it has been implemented as per mockup

"Return cash only" is the new button we added

Actual
![image](https://github.com/user-attachments/assets/80cdee3f-459d-4e90-b4e5-abc08ec3fdaa)

Zeplin
![image](https://github.com/user-attachments/assets/ce61ea14-2f78-4a2b-b7de-859a48e9c0c4)

Reply:
I added the amount in all buttons